clients/lightrider/lightrider-ui-features.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* global ReportUIFeatures */
+
+/**
+ * Extends ReportUIFeatures to use the saved report from a browserified ReportGenerator.
```

Comment:
"saved report" meaning the generic report assets, right? not like a saved report instance of a particular run that's somehow bundled in a way I don't know about :)

Reply:
Uh I think I get what you're asking?  It is just me trying to say that this is used to get report HTML via a report generator and this.json instead of outerHTML like a reg report.